<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>save_current_game_window</class>
 <widget class="QWidget" name="save_current_game_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="close_button">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>265</y>
     <width>300</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Вернуться в меню</string>
   </property>
  </widget>
  <widget class="QFrame" name="crutch">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1901</width>
     <height>971</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>150</y>
      <width>351</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTextEdit" name="save_text_edit">
       <property name="tabStopWidth">
        <number>16</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save_button">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>такое сохранение 
 уже существует</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <zorder>crutch</zorder>
  <zorder>close_button</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
